benchexec/benchexec.py

```diff
@@ -152,6 +152,12 @@ def create_argument_parser(self):
                           metavar="N",
                           help="Limit each run of the tool to N CPU cores (-1 to disable).")
 
+        parser.add_argument("--allowedCores",
+                          dest="coreset", default=None, type=util.parse_int_list,
+                          help="Limit the set of cores BenchExec will use for all runs "
+                               "(Applied only if the number of CPU cores is limited).",
```

Comment:
I actually like this because it means that most users will keep using the normal option for the core limit instead of using this "manual" option. But could you please change it such that benchexec aborts with an invalid-configuration error if this option is given without a core limit? Silently ignoring this option seems very dangerous.